scheduler/util.go

```diff
@@ -335,6 +335,11 @@ func tasksUpdated(a, b *structs.TaskGroup) bool {
 		return true
 	}
 
+	// Check ephemeral disk
+	if !reflect.DeepEqual(a.EphemeralDisk, b.EphemeralDisk) {
+		return true
+	}
```

Comment:
Do you want to remove the disk check below: https://github.com/hashicorp/nomad/pull/1725/files#diff-082e1e20560c8781c7ec6e064125b722R391